src/aws/lambda-encode/src/encodeVideo.js

```diff
@@ -44,17 +44,27 @@ module.exports = async (objectKey, sourceBucket) => {
     const videoInfo = mediainfos.media.track.find(track => track["@type"] === "Video");
     const audioInfo = mediainfos.media.track.find(track => track["@type"] === "Audio");
 
-    if (videoInfo.hasOwnProperty('Height')) {
-      videoSizes = videoSizes.filter(size => size <= videoInfo.Height);
+    if (videoInfo) {
+      if (videoInfo.hasOwnProperty('Height')) {
+        videoSizes = videoSizes.filter(size => size <= videoInfo.Height);
+        if (videoSizes.length === 0) {
+          videoSizes.push(144);
+        }
```

Comment:
Maybe better to keep the sizes definition on top of the page and not hardcode 144 here?
```suggestion
        let usefulVideoSizes = videoSizes.filter(size => size <= videoInfo.Height);
        if (usefulVideoSizes.length === 0) {
          userfulVideoSizes.push(videoSizes[0]);
        }
```